rust/opendp/src/trans/count.rs

```diff
@@ -28,6 +28,24 @@ pub fn make_count<MI, MO, TI>() -> Fallible<Transformation<VectorDomain<AllDomai
         StabilityRelation::new_from_constant(MO::Distance::one())))
 }
 
+
+pub fn make_count_distinct<MI, MO, TI>() -> Fallible<Transformation<VectorDomain<AllDomain<TI>>, AllDomain<MO::Distance>, MI, MO>>
```

Comment:
Huh, I hadn't noticed this before, is there a reason the output carrier is `MO::Distance`? In other places, for dataset-sized things, don't we fix it to `u32`?

Reply:
The reason the output distance is generic is because the geometric mechanism expects the distance and datatype to share the same type. I could make these independent and add an additional cast to the geometric mechanism. I'm not sure if that's better?

Reply:
Ah thanks, yeah probably not worth it.